### DISCUSSION TITLE
Getting Range Error on Cypress Test Runner not via Run mode

### DISCUSSION THREAD

**Discussion by senthiltr** [cypress-io/cypress#17639] [Questions and Help]
HI Team,
        May I know what scenario this range error will come?. Not sure about startoffset option and how the value set for this option. my spec has totally two tests.
I am using page object model and in the test case import two modules and inside these two modules imported some other modules. In this style, when i run my spec file in the test runner, keep on loading tests and login into application and keep on loading test and login into application for near 30 mins. It is not moving to second test. Finally getting the below error in test runner mode.
Error:
The following error originated from your test code, not from Cypress.

  > Start offset -2141824504 is outside the bounds of the buffer

When Cypress detects uncaught errors originating from your test code it will automatically fail the current test.

Cypress could not associate this error to any specific test.

We dynamically generated a new test to display this failure.

But with the same style , in run mode not getting any error. Both tests are executing fine.
Kindly let me know how to resolve Range error.?

My Folder Structure:
Folder Structure and imported style:
Cypress
Lib(For having general function. where ever required it will get call)
  --- a.js
 ----b.js
vLib_po(modules with class and many class variables).
  ---a.po.js
  ----b.po.js(imported a.js, b.js and other .po modules)
Integration
   ----- t1.spec.js- imported a.js, a.po.js, b.po.js
Spec file is executing only when I remove b.po.js or remove imports(a.js, b.js) in b.po.js.
Kindly provide the solution or any comments in this. 

Thanks
Senthilkumar TR

**Comment by mufasa71** [comment]
Looks like out of memory problem

**Reply by senthiltr** [reply]
Thanks For The Reply. :)
The issue not coming when I run using run mode(cli). You mean memory goes for test runner?.
I am having more variables inside class(not inside constructor). Is this cause the object as weighted object?. so memory problem is coming?.
Any solution recommending for memory problem?

Thanks
Senthilkumar TR
